site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -333,7 +335,7 @@ public SecurityAdvice isAllowed(String userId, String function, String reference
 			"-siteInfo-changeAdmin", // 63
 			"-selectAdmin", // 64
 			"-selectAdmin", // 65
-			"-exportMemberList" // 66
+			"-siteInfo-exportContent" // 66
```

Comment:
I think this is still used by `buildUserListPanelContext(VelocityPortlet portlet, Context context, RunData data, SessionState state)` and so it probably needs to go to 67.